manifest.json

```diff
@@ -33,5 +33,8 @@
     "quick-find-search-links": {
       "description": "Search all links"
     }
-  }
+  },
+  "permissions": [
```

Comment:
Thanks! I like the web permissions model in general, even if this is slightly awkward. I'll make these changes, probably tomorrow evening (CET).